packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -145,6 +145,9 @@ done
 %{_sbindir}/foreman-proxy-certs-generate
 
 %changelog
+* Mon Mar 23 2020 Eric D. Helms <ericdhelms@gmail.com> - 1:2.0.0-0.3.rc3
+- Release foreman-installer 2.0.0
```

Comment:
In the tooling I had this at least indicated the RC. Now we have rc1, rc2 and rc3 all looking like it's going GA.

Reply:
Updated manually, this just came from running the scripts.